### DISCUSSION TITLE
betterDefine导入时，.*d.ts如何省略掉 .d 后缀?

### DISCUSSION THREAD

**Discussion by MingRock** [vue-macros/vue-macros#191] [Q&A] [answered]
现在是 import type {IProps} from "@/types/table/props.d";
想实现 import type {IProps} from "@/types/table/props";

**Comment by sxzz** [comment, marked as answer]
Implemented in `unplugin-vue-macros@1.2.2`

**Reply by mattmess1221** [reply]
I get this warning when starting nuxt using @vue-macros/nuxt@0.2.7 and unplugin-vue-macros@1.2.2. `type-fest` is a package made entirely of .d.ts files
```
[vite] warning: unplugin-vue-better-define Error: Failed to resolve entry for package "type-fest". The package may have incorrect main/module/exports specified in its package.json.
  Plugin: unplugin-vue-better-define
```

**Reply by sxzz** [reply]
Could you please raise an issue w/ a reproduction repo?
